mgmt_cli_test.py

```diff
@@ -332,6 +332,50 @@ def test_repair_intensity_feature_on_multiple_node(self):
     def test_repair_intensity_feature_on_single_node(self):
         self._repair_intensity_feature(fault_multiple_nodes=False)
 
+    def test_repair_control(self):
+        InfoEvent(message="Starting C-S write load")
+        self.run_prepare_write_cmd()
+        InfoEvent(message="Flushing")
+        for node in self.db_cluster.nodes:
+            node.run_nodetool("flush")
+        InfoEvent(message="Waiting for compactions to end")
+        self.wait_no_compactions_running(n=90, sleep_time=30)
+        InfoEvent(message="Starting C-S read load")
+        stress_read_thread = self.generate_background_read_load()
+        time.sleep(600)  # So we will see the base load of the cluster
+        InfoEvent(message="Sleep ended - Starting tests")
+        self._create_repair_and_alter_it_with_repair_control()
+        load_results = stress_read_thread.get_results()
+        self.log.info(f'load={load_results}')
+
+    def _create_repair_and_alter_it_with_repair_control(self):
+        keyspace_to_be_repaired = "keyspace2"
+        if not self.is_cred_file_configured:
+            self.update_config_file()
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_repair_control',
+                                               db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self.create_missing_rows_in_cluster(create_missing_rows_in_multiple_nodes=True,
+                                            keyspace_to_be_repaired=keyspace_to_be_repaired,
+                                            total_num_of_rows=292968720)
```

Comment:
still missing a comment of explaining where this 292968720 number is coming from.
it's it % of the whole number of rows ? or what exactly ?

Reply:
Explanation added